sources/api/apiclient/src/main.rs

```diff
@@ -151,6 +152,7 @@ fn usage() -> ! {
             reboot                     Reboots the host.
             exec                       Execute a command in a host container.
             report cis                 Retrieve a Bottlerocket CIS benchmark compliance report.
+            report cis-k8s             Retrieve a Kubernetes CIS benchmark compliance report.
```

Comment:
Shouldn't this command be “conditionality”-compiled based on the variant?

Reply:
I did think about that and decided to just include it. If someone were to try to run the report on an `aws-ecs-*` instance, it would just report back SKIPPED for every assertion.

Conditionally including it makes things slightly more complicated with needing conditional inclusion, especially here with things like documentation. It also changes this CLI "api" by having some commands there on one instance, but not there on others.

It could be done, but I opted to try to keep things as simple as possible. Willing to hear strong opinions though.